server/boot/index.js

```diff
@@ -1,20 +1,52 @@
 /**
- * Module dependencies
+ * External dependencies
  */
-var path = require( 'path' ),
-	build = require( 'build' ),
-	config = require( 'config' ),
-	express = require( 'express' ),
-	morgan = require( 'morgan' ),
-	pages = require( 'pages' );
+import path from 'path';
+import express from 'express';
+import morgan from 'morgan';
+import React from 'react';
+import { renderToStaticMarkup } from 'react-dom/server';
+
+/**
+ * External dependencies
+ */
+import build from 'build';
+import config from 'config';
+import pages from 'pages';
+
+function renderJsxFile( filename, options, callback ) {
```

Comment:
Oh, I didn't realize that the output is already minified. In that case, we're golden!